src/views/AccountInfo/AccountInfoConnectedState.tsx

```diff
@@ -91,6 +93,13 @@ export const AccountInfoConnectedState = () => {
             >
               {stringGetter({ key: STRING_KEYS.DEPOSIT })}
             </Styled.Button>
+            <Styled.IconButton
+              key={DialogTypes.Withdraw}
```

Comment:
nit: `key` is unnecessary here. It is only used when rendering items within a loop.